From 51a7ab21a83eaddb5786233c4992793e273e174a Mon Sep 17 00:00:00 2001
From: Your Name <you@example.com> 
Date: Thu, 17 Aug 2023 14:57:23 +0800
Subject: [PATCH 01/11] io_re_frida_server

---
 server/server.vala | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/server/server.vala b/server/server.vala
index f96c7ca5..62cf0d71 100644
--- a/server/server.vala
+++ b/server/server.vala
@@ -1,7 +1,7 @@
 namespace Frida.Server {
 	private static Application application;
 
-	private const string DEFAULT_DIRECTORY = "re.frida.server";
+	private const string DEFAULT_DIRECTORY = "IOserver";
 	private static bool output_version = false;
 	private static string? listen_address = null;
 	private static string? certpath = null;
-- 
2.39.2 (Apple Git-143)

