From c46e242f768cabe031dfd7968031ea0fcee03ccb Mon Sep 17 00:00:00 2001
From: Your Name <you@example.com> 
Date: Thu, 17 Aug 2023 15:23:06 +0800
Subject: [PATCH 09/11] thread_gdbus

---
 src/anti-frida.py | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/src/anti-frida.py b/src/anti-frida.py
index 1a722f3a..51ef4509 100644
--- a/src/anti-frida.py
+++ b/src/anti-frida.py
@@ -28,4 +28,8 @@ if __name__ == "__main__":
     # thread_gmain
     random_name = "".join(random.sample("abcdefghijklmn", 5))
     log_color(f"[*] Patch `gmain` to `{random_name}`")
-    os.system(f"sed -b -i s/gmain/{random_name}/g {input_file}")
\ No newline at end of file
+    os.system(f"sed -b -i s/gmain/{random_name}/g {input_file}")
+    # thread_gdbus
+    random_name = "".join(random.sample("abcdefghijklmn", 5))
+    log_color(f"[*] Patch `gdbus` to `{random_name}`")
+    os.system(f"sed -b -i s/gdbus/{random_name}/g {input_file}")
\ No newline at end of file
-- 
2.39.2 (Apple Git-143)

