From 54a24b331333f4d5ecc94f991b0f408ee742fa47 Mon Sep 17 00:00:00 2001
From: Your Name <you@example.com> 
Date: Thu, 17 Aug 2023 15:25:47 +0800
Subject: [PATCH 11/11] protocol_unexpected_command

---
 src/droidy/droidy-client.vala | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/droidy/droidy-client.vala b/src/droidy/droidy-client.vala
index 0ed2edeb..5ab4c006 100644
--- a/src/droidy/droidy-client.vala
+++ b/src/droidy/droidy-client.vala
@@ -1013,7 +1013,7 @@ namespace Frida.Droidy {
 						case "OPEN":
 						case "CLSE":
 						case "WRTE":
-							throw new Error.PROTOCOL ("Unexpected command");
+							break;
 
 						default:
 							var length = parse_length (command_or_length);
-- 
2.39.2 (Apple Git-143)

